static-build/nemu/Dockerfile

```diff
@@ -40,20 +41,20 @@ RUN apt-get install -y \
 	    zlib1g-dev
 
 RUN cd  .. && git clone --depth=1 "${NEMU_REPO}" nemu
-RUN git checkout "${NEMU_VERSION}"
+RUN git fetch origin --tags && git checkout "${NEMU_VERSION}"
 RUN git clone https://github.com/qemu/capstone.git capstone
 RUN git clone https://github.com/qemu/keycodemapdb.git ui/keycodemapdb
 
 ADD configure-hypervisor.sh /root/configure-hypervisor.sh
 
-RUN PREFIX=/opt/kata /root/configure-hypervisor.sh -s kata-nemu | xargs ./configure \
+RUN PREFIX="${PREFIX}" /root/configure-hypervisor.sh -s kata-nemu | xargs ./configure \
        --with-pkgversion=kata-static
 
 RUN make -j$(nproc)
 RUN make install DESTDIR=/tmp/nemu-static
 
-RUN wget "${NEMU_OVMF}" && mv OVMF.fd /tmp/nemu-static/opt/kata/share/kata-nemu/
-RUN mv /tmp/nemu-static/opt/kata/bin/qemu-system-x86_64 /tmp/nemu-static/opt/kata/bin/nemu-system-x86_64
-RUN wget "${VIRTIOFSD_RELEASE}/${VIRTIOFSD}" && chmod +x ${VIRTIOFSD} && mv ${VIRTIOFSD} /tmp/nemu-static/opt/kata/bin/
+RUN wget "${NEMU_OVMF}" && mv OVMF.fd /tmp/nemu-static/"${PREFIX}"/share/kata-nemu/
+RUN mv /tmp/nemu-static/"${PREFIX}"/bin/qemu-system-x86_64 /tmp/nemu-static/"${PREFIX}"/bin/nemu-system-x86_64
```

Comment:
I appreciate you're changing what was already there, but this is a bug: NEMU is supported on arm64 too:

- https://github.com/intel/nemu#architectures

Any chance you could add a commit before this one to set the architecture here from a variable defined at the top?

Reply:
Currently building static packages of arm64 is not working on my tests. That is definitely in the to-do list of things as a follow on just to unblock this build and make sure we test nemu + virtiofs the same way we ship. I will have a bug opened for it and work on that right after this.